lib/Containers/MainContainer.js

```diff
@@ -9,7 +8,7 @@ export default class MainContainer extends PureComponent {
   render() {
     return (
       <div>
-        <Header type="h2">Wallet Dashboard</Header>
+        <h2>Wallet Dashboard</h2>
```

Comment:
Why not using bpanel-ui's Header? Using bpanel-ui components ensures it inherits the theme styles.

Reply:
This is just a test

Reply:
Just wondering since this commit removed it.